### PR TITLE
Remove usage of Buildkite.

### LINKED ISSUE #7493: Migrate off BuildKite
We're ramping down our BuildKite usage and this project is one of the few left on it. How much work would it be to migrate to GH Actions/similar so that CI was under your span of control?

### PR DESCRIPTION
**Pull request by bmarty**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [x] Technical
- [ ] Other :

## Content

Remove usage of Buildkite.
Build number is just removed.
Related script will need to be updated separately.

## Motivation and context

Closes #7493 

Release script will have to be updated for the next release, but it's not part of the project file.

TODO:
- [ ] Update topic of https://matrix.to/#/#element-android:matrix.org

## Tests

- NA

## Tested devices

- [ ] Physical
- [ ] Emulator
- OS version(s):

## Checklist

- [ ] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [ ] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release.yml [hunk 1]
```diff
@@ -40,8 +40,8 @@ body:
         - [ ] Increase version (versionPatch + 2) in `./vector/build.gradle`
         - [ ] Change the value of SDK_VERSION in the file `./matrix-sdk-android/build.gradle`
         - [ ] Commit and push `develop`
-        - [ ] Wait for [Buildkite](https://buildkite.com/matrix-dot-org/element-android/builds?branch=main) to build the `main` branch.
-        - [ ] Run the script `~/scripts/releaseElement.sh`. It will download the APKs from Buildkite check them and sign them.
+        - [ ] Wait for the [GitHub action](https://github.com/vector-im/element-android/actions/workflows/build.yml?query=branch%3Amain) to build the `main` branch.
+        - [ ] (TODO This script has to be updated) *Run the script `~/scripts/releaseElement.sh`. It will download the APKs from Buildkite check them and sign them.*
```
Comment (mnaturel) [inline .github/ISSUE_TEMPLATE/release.yml:44]: Out of curiosity, where is this `releaseElement.sh` script?

File: tools/install/installFromBuildkite.sh [hunk 2]
```diff
@@ -3,6 +3,9 @@
 # Exit on any error
 set -e
 
+echo "Sorry, this script needs to be updated to download APKs from GitHub action. Buildkite is not building APKs anymore."
```
Comment (mnaturel) [inline tools/install/installFromBuildkite.sh:6]: Will the script be updated in another dedicated PR?

**Review by mnaturel — COMMENTED** [review]
Right now I have this for local build in the splashscreen:



I don't know if this is already the case but, in the case of a CI build, do you know if it would be possible to add the commit hash after the branch name to have more precision about where we are in the branch?

File: tools/install/installFromBuildkite.sh (hunk 2 above)
Comment (bmarty) [reply tools/install/installFromBuildkite.sh:6]: I guess I will write another script to install from GitHub, but yes, in another PR. I will write it during the next release.

Comment (bmarty) [reply tools/install/installFromBuildkite.sh:6]: (Actually I am working on it right now)

File: .github/ISSUE_TEMPLATE/release.yml (hunk 1 above)
Comment (bmarty) [reply .github/ISSUE_TEMPLATE/release.yml:44]: @mnaturel https://github.com/vector-im/element-android/pull/7583/commits/f75214648eeb853dcd3df226a9961c8498be0452

Can you check the script on your side?

**Comment by mnaturel** [comment]
@bmarty When testing a build from this PR using the script, I get this info on the splashscreen:



We have `HEAD` for the branch name but I don't know why. And I cannot find the hash commit `bd62fbed` in the repo. Do you have an idea of what is happening?
